examples/ezstream-full.xml

```diff
@@ -110,7 +110,13 @@
     <!-- Program to query for metadata instead of from media files -->
     <!-- <program>metadata.sh</program> -->
 
-    <!-- Metadata format -->
+    <!-- 
```

Comment:
I reverted the comment back to original concise format and updated the doc/ezstream.1.in.in with the album field information.  I left the defaults alone and just added the area that describes the placeholder and where its located in the config. I copied the format of the artist and title placeholder descriptions.